src/platform/datapath_raw_socket.c

```diff
@@ -734,6 +734,9 @@ CxPlatDpRawParseEthernet(
         return;
     }
 
+    CxPlatCopyMemory(&Packet->Route->LocalLinkLayerAddress, Ethernet->Destination, sizeof(Ethernet->Destination));
+    CxPlatCopyMemory(&Packet->Route->NextHopLinkLayerAddress, Ethernet->Source, sizeof(Ethernet->Source));
```

Comment:
Before this change, we leave LocalLinkLayerAddress and NextHopLinkLayerAddress being zeros. But then we will overwrite (excluding Route->State) route in the connection that the datagram gets indicated to? Wouldn't that just overwrite local and next hop mac addresses to zeros and end up causing the connection sending frames with zero'd MAC addresses?

Reply:
Oh, I see. Overwriting route info only happens once per path for (stateful) connections.

Reply:
Resolved offline